### PR TITLE
Portal links for Azure SQLDB added for 1.22 release

### PR DESCRIPTION
**Pull request by smartguest**
This adds Azure portal links for the deployment of databases, with a button that changes in text when a user selects a database.

### CODE REVIEW

File: extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts [hunk 1]
```diff
@@ -176,6 +176,14 @@ export class ResourceTypePickerDialog extends DialogBase {
 
 	private selectResourceType(resourceType: ResourceType): void {
 		this._selectedResourceType = resourceType;
+		//handle special case when resource type is SQL Azure Database
+		if (this._selectedResourceType.name === 'sql-azure-setup') {
```
Comment (Charles-Gagnon) [inline extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts:180]: I'd rather see this be an optional config field in the JSON (defaulting to Select if nothing is provided)

**Review by aasimkhan30 — APPROVED** [review]

**Review by abist — APPROVED** [review]

**Comment by aasimkhan30** [comment]
@smartguest can you merge these changes into main first?

**Comment by Charles-Gagnon** [comment]
@smartguest I'm going to close this - you should merge this into main first and then cherry pick into the release branch so the commit hashes stay the same.

**Comment by abist** [comment]
I think we can just switch the target branch to main here

**Comment by Charles-Gagnon** [comment]
He already opened one for main so I figured he'd want to just use that one 😄 https://github.com/microsoft/azuredatastudio/pull/12322

**Comment by smartguest** [comment]
Switching to main led to some other files being changed along with these ones, hence the new PR.

**Pull request closed without merging**